Sources/CubeFoundationSwiftUI/Misc/ResourceName.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 
 public protocol ResourceName: ExpressibleByStringLiteral, Hashable {
-    var string: String { get set }
```

Comment:
What impact does this have elsewhere? 🤔

Reply:
The package builds fine with the changes in this PR, for the font it was handled by the switch. Don't think it will break user's of the protocol